### ISSUE TITLE
Review how markers/icons are visualised on map

### ISSUE THREAD

**Issue by ColmMassey** [SolidarityEconomyAssociation/owned-by-oxford-project#9]
We currently have simple options for how we display markers and icons on markers. We need design a way of configuring more flexible options. It needs to cope with data from multiple sources, and what happens when different combinations of source data are selected.

We may also want to extend it to include highlighting. We should start with a few stories.

**Comment by ColmMassey** [comment]
Story 1:
OBO wants to be able to illustrate that some of the projects/organisations displayed on the public map have 'joined' the OBO initiative. ALL initiatives on the map follow the same colour coding and choice of icons, but 'non-members' have a much more washed out/slightly translucent colouring.

**Comment by ColmMassey** [comment]
@MelissaMc20

**Comment by ColmMassey** [comment]
Duplicate of https://github.com/SolidarityEconomyAssociation/sea-map/issues/98 and
https://github.com/SolidarityEconomyAssociation/sea-map/issues/107

**Comment by ColmMassey** [comment]
Story 2: The Mersey green project have commissioned an artist to create some icons to identify the location of an initiative on their sea-maps, based on the _Primary Activities_ of an initiative . (https://leafletjs.com/examples/custom-icons/) They tell us how they want them to be mapped to each activity. We use them to replace both the currently used marker and icon.

**Comment by ColmMassey** [comment]
Story 3: We begin publishing the fact that a Dotcoop registrant has also registered to use the coop marque. We would like to highlight the markers of all these dotcoop registrants.

**Comment by ColmMassey** [comment]
If the config file we need to be able to define some simple rules which informs the UI which marker type,colour,icon, shoadowing etc should be used. For OBO I expect this will need to be something like

If initiative is not from owned-by-oxford dataset then reduce set saturation level of marker colour to 50% or
If initiative has a Relationship specified and it is one of the following (SteeringCOmmittee/Anchor Institution...) then us this particular marker.

**Comment by ColmMassey** [comment]
Adding @wu-lee to the ticket as needs coder input to practicalities of how flexible this can be.

**Comment by wu-lee** [comment]
Miro design board: https://miro.com/app/board/o9J_ltkVpc4=/ (Over to the right of this board.)

**Comment by MelissaMc20** [comment]
https://xd.adobe.com/view/2f1ee132-c6fe-4355-967a-3ca4c363bfc9-7a4d/

**Comment by wu-lee** [comment]
Just a note that we may need to be able to select different vocabs on the panel, and this should be incorporated in the plan

**Comment by ColmMassey** [comment]
> Miro design board: https://miro.com/app/board/o9J_ltkVpc4=/

Is this the right miroboard? Can you create a link zooming to the right part of the board?

**Comment by ColmMassey** [comment]
With the current icon choices and size of screen/marker etc, it is not possible to see the icons.

Are we assuming that the icons are always white or at least monochrome?

Can you take a look at https://www.greenmap.org/make-map/green-map-icons @MelissaMc20 

What options do we have for using icons from here?

**Comment by wu-lee** [comment]
> > Miro design board: https://miro.com/app/board/o9J_ltkVpc4=/
> 
> Is this the right miroboard? Can you create a link zooming to the right part of the board?


Um... That was me trying to capture information prior to talking about it in last week's meeting.  I confess to finding Miro boards disorienting, and that I'm not certain it is the right one?

**Comment by ColmMassey** [comment]
> I confess to finding Miro boards disorienting,

I agree. I think we should not create them as large as we do, and we need to develop methodolgy for naming and 'filing' them.

**Comment by ColmMassey** [comment]
> Miro design board: https://miro.com/app/board/o9J_ltkVpc4=/

I think this is a more useful link 
https://miro.com/app/board/o9J_ltkVpc4=/?moveToWidget=3458764521614023183&cot=14

**Comment by MelissaMc20** [comment]
@ColmMassey - yes, I'd say icons inside markers should always be some sort of monochrome, white is the more aesthetically pleasing colour, with grey icon colour change depending on selected state.

I'm happy to consider markers from other sources, or abandoning the actual marker pin and only using icons could also work. I used the markers from Awesome-Marker as @wu-lee advised using multiple sources for markers might be more difficult. But I think it's still open to interpretation. 

The actual style of the marker is also open to any suggestions, the mock-up created is more of a conceptual mock-up, to show how the colours would work for multiple datasources.

**Comment by wu-lee** [comment]
I think the next step here is just to pick some sort of a scheme and show that it works.  It doesn't so much matter what it is.  

But that discussion should probably live in issue https://github.com/SolidarityEconomyAssociation/sea-map/issues/128 - I'll post there again with an update when there's something to review.

**Comment by ColmMassey** [comment]
I suggest practicing on the solidarity oxford map, as it is 'ours'.

**Comment by ColmMassey** [comment]
We are now evaluating a project for the Keep it in the Communkity Project. See https://docs.google.com/document/d/1zFwu1aYlR61DtiOwHs8EtBIz6DK2MEeGhcDSypq24_I/edit

Can you @MelissaMc20 @wu-lee think of any reason why we couldn't implement this? 24 seems like a lot of catagories. I recall there was some restriction on how many different icons one could have in a set. Did I remeber that correctly?

**Comment by ColmMassey** [comment]
Let's test new method with the KIITC project https://github.com/DigitalCommons/KIITC/issues/1

**Comment by ColmDC** [comment]
We have a new requirement coming soon, to distinguish co-ops from credit unions on the dotcoop map.
